frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -358,21 +358,23 @@ const Card: React.FC<CardProps> = ({ panel }) => {
 
   const panelClassModifier = getPanelClassModifier(panel);
   return (
-    <div
-      className={`monitoring-dashboards__panel monitoring-dashboards__panel--${panelClassModifier}`}
-    >
-      <DashboardCard
-        className="monitoring-dashboards__card"
-        gradient={panel.type === 'grafana-piechart-panel'}
+    panel.type !== 'gauge' && (
```

Comment:
We should generalize this so that we hide any panel type we don't recognize.

Reply:
All set!